airflow/timetables/events.py

```diff
@@ -0,0 +1,111 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import itertools
+from typing import Iterable, Optional
+
+import pendulum
+from pendulum import DateTime
+
+from airflow.timetables.base import DagRunInfo, DataInterval, TimeRestriction, Timetable
+
+
+class EventsTimetable(Timetable):
+    """
+    Timetable that schedules DAG runs at specific listed datetimes. Suitable for
+    predictable but truly irregular scheduling such as sporting events.
+
+    :param event_dates: List of datetimes for the DAG to run at. Duplicates will be ignored.
+    :param restrict_to_events: Whether manual runs should use the most recent event or
+    the current time
```

Comment:
```suggestion
        the current time
```

should fix the docs build